types/koa-useragent/index.d.ts

```diff
@@ -0,0 +1,74 @@
+// Type definitions for koa-useragent 1.2
+// Project: https://github.com/rvboris/koa-useragent
+// Definitions by: Boris Ryabov <https://github.com/rvboris>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+import Koa = require("koa");
+
+declare namespace UserAgent {
+    interface UserAgent {
```

Comment:
It looks like this interface is exposed to the user, so it would be worth having some tests the exercise this (have a class that implements this interface or a function that takes this as an argument). If it's not meant to be consumed directly and only through the Context, then the interface should not be exported.